src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java

```diff
@@ -75,6 +75,7 @@ public void start(
         if (javaOptions != null && javaOptions.size() > 0) {
             args.addAll(javaOptions);
         }
+        args.add("-Djava.net.preferIPv4Stack=true");
```

Comment:
Do you verify those properties in a test?

Reply:
I was, but we decided to remove that test with @leszko since because of the environment isolation issues.